eliot-service/eliot/symbolicate_resource.py

```diff
@@ -295,25 +280,26 @@ def parse_sym_file(self, debug_filename, debug_id, data):
                 tags=[f"reason:{psfe.reason_code}"],
             )
 
-    def get_symcache(self, module_info, debug_stats):
+    def get_symcache(self, debug_filename, debug_id, debug_stats):
         """Gets the symcache for a given module.
 
         This uses the cachemanager and downloader to get the symcache. It modifies
         the Module in place.
 
-        :arg module_info: the ModuleInfo data class
+        :arg debug_filename: the debug filename
+        :arg debug_id: the debug id
         :arg debug_stats: DebugStats instance for keeping track of timings and other
             useful things
 
-        """
-        debug_filename = module_info.debug_filename
-        debug_id = module_info.debug_id
+        :returns: symcache, filename or None
```

Comment:
I hate this return, but I didn't have a better way to deal with this well without creating data structures.